package.json

```diff
@@ -129,6 +130,18 @@
           "typescript",
           "webpack"
         ]
+      },
+      {
+        "packages": [
+          "@fluentui/accessibility",
+          "@fluentui/react-bindings",
+          "@fluentui/react-northstar"
+        ],
+        "dependencies": [
```

Comment:
there are bunch missing pieces related to this change around webpack/jest/rollup configs. Elizabeth is addressing in PR #15644